appcelerator/titanium_mobile
# [TIMOB-25773] Android: Fixed app build failure caused by " -" in project path

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25773

**Summary:**
- This issue was introduced in Titanium 7.0.2.
- Now double quotes paths added to ProGuard configuration file.
- Now double quotes paths passed to gradle shell/batch file.
  * Needed on Windows. Not required on Mac, but still works.
- Also escapes double quotes in file/folder names since that is legal on Mac and Linux. (Not allowed on Windows.)

**Test:**
1. Go to a Mac machine.
2. Create a "Classic" Titanium app named "Dash -Test". _(Note the space and dash.)_
3. Make sure the project folder name is also "Dash -Test".
4. Build for Android.
5. Verify that the APK is successfully created.
6. Repeat steps 2-5 on Windows.
